packages/fungible-token/bridge-fungible-token/tests/utils/environment.rs

```diff
@@ -200,9 +206,10 @@ pub async fn setup_environment(
             &message_sender.into(),
             predicate_root,
             msg.0,
-            message_nonce.into(),
+            message_nonce,
             msg.1.clone(),
-        ))
+        ));
+        message_nonce[0] += 1;
```

Comment:
Just adding a note because I'm unsure if this is an issue. I believe this will only go up to the `u8` limit so if we ever create (if we even can create that many) more than the limit then this may make the entire suite fail.

Perhaps we should add a comment stating this in case this ever becomes an issue

Reply:
Shouldn't `message_nonce[0]` only overflow at 2^64? `message_nonce` is really just 4 64bit number, right?

Reply:
I believe it's [u8; 32] under the hood